README.md

```diff
@@ -82,7 +82,7 @@ When developing with multiple repositories, you may need to link packages locall
      {
        "dependencies": {
          "@example/package-name": "<path-to-test-data-package>",
-         "@commercetools-test-data/commons": "../../../test-data/models/commons"
+         "@commercetools-test-data/commons": "../../../test-data/standalone/src/models/commons"
```

Comment:
Actually, this is not valid since the `@commercetools-test-data/commons` package does not exist anymore.
We now need to replace it with the new single package (`@commercetools/composable-commerce-test-data`) in the README.

Reply:
fixed in: b02bf8b02d41a325a35f359de5555c514328a6cb